model/host/db.go

```diff
@@ -457,3 +457,35 @@ func QueryWithFullTaskPipeline(match bson.M) []bson.M {
 		},
 	}
 }
+
+type InactiveHostCounts struct {
+	HostType string `bson:"_id"`
+	Count    int    `bson:"count"`
+}
+
+func InactiveHostCountPipeline() []bson.M {
```

Comment:
does this need to be public or can we wrap the db.Aggregate call in a single public function

Reply:
Made private